src/data-reporting-quicksight-stack.ts

```diff
@@ -174,6 +174,13 @@ export class DataReportingQuickSightStack extends Stack {
     cr.node.addDependency(vPCConnectionResource);
     cr.node.addDependency(template);
 
+    const userCustomResource = createInternelUserCustomResource(this, {
```

Comment:
Is it duplicated with the work in #114 ? @tyyzqmf

Reply:
not duplicate with work in #114

Reply:
This user is preferably created by backend when the pipeline is creating, if QuickSight is enabled for the pipeline.